ansible/roles/vm_set/tasks/remove_topo.yml

```diff
@@ -8,6 +8,13 @@
   when: ptf_imagename is defined and ptf_imagename == "docker-keysight-api-server"
 
 - block:
+
+  - name: Stop mux simulator
+    include_tasks: control_mux_simulator.yml
+    vars:
+      mux_simulator_action: stop
+    when: "'dualtor' in topo"
```

Comment:
We need to be cautious to remove mux simulator server when remove topo. Because several topos may share the same mux simulator server instance.

Reply:
we do not need to share mux simulator for different topology.

Reply:
Thanks for the comments!
To start different mux simulator for each topology, we need to specify a tcp port for each topology. The chance of TCP port used by other services is higher.
I have pushed a new commit to address the issues:
* Make the mux simulator a systemd service.
* While add-topo, record the current vm_set name in a text file. Start mux-simulator service.
* While remove-topo, remove the current vm_set name from the text file. If the text file is empty, it means that no other setup is using the mux-simulator service, then stop the mux-simulator service.